ShopGo/NavigationManager.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  NavigationManager.swift
+//  ShopGo
+//
+//  Created by Quratulain on 8/5/23.
+//
+
+import UIKit
+
+enum AppScreen {
+    case LoginSignup
+    case Onboarding
+    case CreateAccount
+    case Login
+    case Terms
+    case Home
+    case MyProfile
+    case OTP
+    
+    var viewControllerIdentifier: String {
+        switch self {
+        case .LoginSignup:
+            return "LoginSignupViewController"
+        case .Onboarding:
+            return "OnBoardingPageViewController"
+        case .CreateAccount:
+            return "CreateAccountViewController"
+        case .Login:
+            return "LoginViewController"
+        case .Terms:
+            return "TermsViewController"
+        case .Home:
+            return "HomeViewController"
+        case .MyProfile:
+            return "MyprofileViewController"
+        case .OTP:
+            return "OTPViewController"
+        }
+    }
+}
+
+class NavigationManager {
+    func navigate(to screen: AppScreen, in storyboardName: String,from viewController: UIViewController) {
+        
+        let destinationViewController: UIViewController
```

Comment:
Remove this.

Reply:
We cannot remove this as I have commented the reason in the comment bellow